slices/binutils.yaml

```
@@ -0,0 +1,26 @@
+package: binutils
+
+essential:
+  - binutils_copyright
+
+slices:
+  assembler:
+    # Note: `v3-essential` only works for chisel>=1.3.0
+    v3-essential:
+      binutils-aarch64-linux-gnu_assembler: {arch: [arm64]}
+      binutils-x86-64-linux-gnu_assembler: {arch: [amd64]}
+    contents:
+      /usr/bin/as:  # Symlink to ${ARCH_TRIPLET}-as
```

In line with what I proposed for `rustc` we could start flagging symlinks like this:

```suggestion
      /usr/bin/as:  # -> ${ARCH_TRIPLET}-as
```